### PR TITLE
feat: add `versions` to output versions of AsyncAPI tools used (#361)

### PR DESCRIPTION
**Pull request by aeworxet**
This PR adds to CLI command

```
asyncapi config versions
```
which outputs versions of AsyncAPI tools used, in form
```
@asyncapi/cli/0.44.2 linux-x64 node-v16.17.0
  ├@asyncapi/avro-schema-parser/3.0.0
  ├@asyncapi/bundler/0.3.11
  ├@asyncapi/converter/1.2.0
  ├@asyncapi/diff/0.4.1
  ├@asyncapi/generator/1.10.0
  ├@asyncapi/modelina/1.6.1
  ├@asyncapi/openapi-schema-parser/3.0.0
  ├@asyncapi/optimizer/0.1.18
  ├@asyncapi/parser/2.0.0
  ├@asyncapi/raml-dt-schema-parser/4.0.0
  └@asyncapi/studio/0.17.3

Repository: https://github.com/asyncapi/cli
```

Resolves https://github.com/asyncapi/cli/issues/361

### CODE REVIEW

File: src/commands/config/versions.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+export default class Versions extends Command {
+  static description = 'Show versions of AsyncAPI tools used';
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+  };
+
+  private dependencies: string[] = [];
+  private dependency = '';
```
Comment (derberg) [inline src/commands/config/versions.ts:12]: just curious why we need those outside `run` if they are anyway private and needed only inside the scope of the run?

**Review by derberg — COMMENTED** [review]
LGTM, just one question left

File: src/commands/config/versions.ts (hunk 1 above)
Comment (aeworxet) [reply src/commands/config/versions.ts]: Yep. Got carried away by OOP. Changed.

**Review by derberg — APPROVED** [review]

**Pull request merged**